lib/matplotlib/backend_bases.py

```diff
@@ -1164,7 +1165,7 @@ def get_sketch_params(self):
         """
         return self._sketch
 
-    def set_sketch_params(self, scale=None, length=None, randomness=None):
+    def set_sketch_params(self, scale=None, length=128, randomness=16):
```

Comment:
This is a slight change of behavior in that anything 'falsy' would give the default behavior.  My guess is that this was not intended.

This came in via https://github.com/matplotlib/matplotlib/pull/1329 which has lots of discussion around it.